src/metadata/taglib_handler.cc

```diff
@@ -419,7 +419,7 @@ std::unique_ptr<IOHandler> TagLibHandler::serveContent(const std::shared_ptr<Cds
         if (!f || !f->isValid() || !f->tag())
             throw_std_runtime_error("Could not open file {} as OGG", itemLocation);
 
-        const TagLib::List<TagLib::FLAC::Picture*> picList = reinterpret_cast<TagLib::Ogg::XiphComment*>(f->tag())->pictureList();
+        const auto picList = static_cast<TagLib::Ogg::XiphComment*>(f->tag())->pictureList();
```

Comment:
From the spec `static_cast` produces extra code to align the pointers. Do you see the need for this.

Reply:
Removed.

Reply:
So clang-tidy actually warns on this: https://clang.llvm.org/extra/clang-tidy/checks/cppcoreguidelines/pro-type-static-cast-downcast.html

looks like dynamic_cast is the proper thing to use here.